src/SmartComponents/Applications/Applications.js

```diff
@@ -15,36 +15,43 @@ export const getAppId = ({ params } = {}) => {
 const Applications = ({ appsConfig, saveValues, match, getSchema, getConfig, configLoaded, loaded, schema }) => {
     const currApp = appsConfig && appsConfig[getAppId(match)] || getAppId(match);
     const appName = ((currApp.frontend && currApp.frontend.title) || currApp.title) || currApp;
+    const [ isOrgAmin, setIsOrgAdmin ] = useState(false);
 
     useEffect(() => {
         register(reducers);
         register({ notifications });
         if (!appsConfig) {
             getConfig();
         }
+
+        insights.chrome.auth.getUser().then((user) => setIsOrgAdmin(!user.identity.user.is_org_admin));
     }, []);
 
     useEffect(() => {
         if (currApp && typeof currApp !== 'string') {
             getSchema(currApp?.api?.apiName || match.params.id, currApp.api);
         }
     }, [ currApp ]);
+
     return (
         <React.Fragment>
-            <PageHeader>
-                <PageHeaderTitle title='Applications settings'/>
-                {
-                    configLoaded ?
-                        <p>{ `Settings for ${ appName}` }</p> :
-                        <Skeleton size='sm' />
-                }
-            </PageHeader>
+            { configLoaded &&
```

Comment:
Is there a reason why we don't want a skeleton here?

Reply:
Good point! That was a bit tricky since the appName is before loading of the scheme lowercase and after it uppercase. My first solution was to hide the header completely during loading so I removed the skeleton. 
Now I pushed a second version where I display the header with the skeleton as it was before and I am making the first letters of appName uppercase manually in the template. I cannot modify the appName property since it is being used for fetching the scheme.
Do you have a better solution? Not sure if it's the best practice